packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapJSONConversions.m

```diff
@@ -141,4 +141,34 @@ + (nullable GMSCameraUpdate *)cameraUpdateFromChannelValue:(NSArray *)channelVal
   }
   return nil;
 }
+
++ (NSArray<GMSStrokeStyle *> *)strokeStylesFromPatterns:(NSArray *)patterns
+                                            strokeColor:(UIColor *)strokeColor {
```

Comment:
is this function tested?

Reply:
Had left a [comment](https://github.com/flutter/packages/pull/5757#issuecomment-1876560460) above about why I think this cant be tested

Reply:
@hellohuanlin Is there any next step on this?

Reply:
It would be good to have a test that you get back an array with the expected number of elements (and a comment explaining why that's all it's testing). It should include both gap and non-gap elements. That will at least exercise all of the codepaths here, so that, e.g., if one of them had a bug that caused an exception, we'd catch that.

Reply:
Alright!